### DISCUSSION TITLE
How to use milvus python sdk if I use url instead of host and port

### DISCUSSION THREAD

**Discussion by bitxinyu** [milvus-io/milvus#4519] [Q&A and General discussion]
for example:
My colleague give me a url (http://abc.def.com/milvus-service) which has been mapping to server port 19530.
Thus, how to use milvus python sdk to connect Milvus server?
I have test `milvus= Milvus(url="http://abc.def.com/milvus-service")` , but failed

**Comment by BossZou** [comment]
@bitxinyu , which version of milvus and pymilvus you used? You may try `milvus= Milvus(url="tcp://{domain name}:{port}")`.

**Reply by bitxinyu** [reply]
Hi, I mean `http://abc.def.com/milvus-service == {host}:{port}` , The domain name already represents host+port
milvus 0.11.0
pymilvus 0.3.0

**Reply by shiyu22** [reply]
@bitxinyu , pymilvus 0.3.0 don't support http protocol.

**Reply by bitxinyu** [reply]
> @bitxinyu , pymilvus 0.3.0 don't support http protocol.

Thus, I can only use restful api, right?

**Reply by BossZou** [reply]
yes.

**Reply by bitxinyu** [reply]
sad...
Thanks anyway
